Pipfile

```diff
@@ -5,24 +5,25 @@ verify_ssl = true
 
 [dev-packages]
 pipenv-setup = {editable = true,path = "."}
-pytest = {markers = "python_version>='3.6'",version = "~=5.2"}
-pytest-mypy = {markers = "python_version>='3.5'",version = "~=0.3"}
-pytest-cov = "~=2.7"
-pytest-datadir = "~=1.3"
-pytest-xdist = "~=1.29"
-tox = "~=3.14"
-autopep8 = "~=1.4"
+pytest = "*"
+pytest-mypy = "*"
+pytest-cov = "*"
+pytest-datadir = "*"
+pytest-xdist = "*"
+tox = "*"
+autopep8 = "*"
 
 [packages]
-pipfile = "~=0.0"
-black = {markers = "python_version>='3.6'",version = "==19.10b0"}
-colorama = "~=0.4"
-packaging = "~=20.0"
-requirementslib = "~=1.5"
-vistir = "~=0.4"
-autopep8 = {markers = "python_version<'3.6'",version = "~=1.4"}
-six = "~=1.12"
-typing = {markers = "python_version<'3.7'", version = "~=3.7"}
+pipfile = "*"
+black = {markers = "python_version>='3.6'",version = "==21.9b0"}
+colorama = "*"
+packaging = "*"
+requirementslib = "*"
+vistir = "*"
+autopep8 = "*"
+six = "*"
+typing = "*"
+chardet = "*"
```

Comment:
Where you able to check if any of these packages may have been pinned for a reason?